CONTRIBUTING.md

```diff
@@ -50,13 +46,36 @@ working directly on your master branch.
     $ git push origin <branch-name>
     ```
 
-2. Write a unit test for your code (optional)
+3. Write a unit test for your code (optional)
 
     OpenQuestion uses `pytest` for unit testing. To run the test suite,
     first navigate to the "tests" directory then use the `pytest` command from your terminal.
 
-3. Submit a pull request (PR) to merge your new branch to OpenQuestion's master branch
+4. Submit a pull request (PR) to merge your new branch to OpenQuestion's master branch
 
     For details on creating a PR see GitHub documentation [Creating a pull
     request](https://help.github.com/en/articles/creating-a-pull-request). 
 
+
+### Hacking
```

Comment:
Is there a better word than "Hacking"?

How about "Working with OpenQuestion locally"